hosts

```diff
@@ -119,7 +119,7 @@ lib-approvals-staging1.princeton.edu
 [approvals_prod]
 lib-approvals-prod1.princeton.edu
 [percona_staging_cluster]
-maria-staging1.princeton.edu
+#maria-staging1.princeton.edu
```

Comment:
Instead of commenting out the items that are still in the cluster machines we should create a new set of hosts percona_staging & percona_prod and move the host that will not be clustered there.